lib/query-interface.js

```diff
@@ -497,10 +497,13 @@ QueryInterface.prototype.removeIndex = function(tableName, indexNameOrAttributes
 };
 
 QueryInterface.prototype.insert = function(instance, tableName, values, options) {
+  options = options || {};
+  options.hasTrigger = !!(instance && instance.__options && instance.__options.hasTrigger);
```

Comment:
instance is undefined in https://github.com/sequelize/sequelize/blob/master/test/integration/query-interface.test.js#L210, the code in my commit handles that gracefully